FirebaseAI/Sources/AppCheckOptions.swift

```diff
@@ -0,0 +1,48 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Configurable options for how App Check is used within a ``FirebaseAI`` instance.
+///
+/// Can be set when creating a ``FirebaseAI.Config``.
+@available(iOS 15.0, macOS 12.0, macCatalyst 15.0, tvOS 15.0, watchOS 8.0, *)
+public struct AppCheckOptions: Sendable, Hashable, Encodable {
```

Comment:
`Sendable` conformance looks good. Why the `Hashable` and `Encodable` conformance?

Reply:
`Hashable` is so that is can be used within `InstanceKey`--since that conforms to `Hashable`--but `Encodable` isn't required afaik, I'm not sure why I have it there. Maybe the IDE auto added it when I was adding `Hashable`, but I've removed it.

Reply:
Okay, SGTM. Thanks for explanation.